### PR TITLE
Fixed git url and added new line for pre-build-service-job

### PR DESCRIPTION
**Pull request by bamachrn**
* Pre-build-service-job was not creating jobs in ci.centos.org due to error in template processing.
* updated git url in job template to point the rendered repo

### CODE REVIEW

**Comment by bamachrn** [comment]
#dotests

File: pre_build/cccp_index_pre_build_check.py [hunk 1]
```diff
@@ -26,7 +27,9 @@ def projectify(
     """
     Projectifying container-index entry to be used for generating job template.
 
-    This function puts all the parameter from container-index entry to a dictory named project. this dictionary later used for rendering the job template.
+    This function puts all the parameter from container-index entry to a
+    dictory named project. this dictionary later used for rendering the job
```
Comment (navidshaikh) [inline pre_build/cccp_index_pre_build_check.py]: typo: dictory --> dictionary

typo: named --> namely

typo .this dictory --> .This dictionary

File: pre_build/cccp_index_pre_build_check.py [hunk 2]
```diff
@@ -115,7 +119,9 @@ def main(indexdlocation):
     """
     Conververting index entries to jenkins job configs.
 
-    This function takes project dictionary generated from container-index as input, renders it to the prebuild-job config and creates jobtemplates to be used for creating the pre-build jobs in ci.centos.org.
+    This function takes project dictionary generated from container-index as
+    input, renders it to the prebuild-job config and creates jobtemplates to
```
Comment (navidshaikh) [inline pre_build/cccp_index_pre_build_check.py]: typo: *job templates

**Comment by navidshaikh** [comment]
oh! The PR is already merged, I didn't realize it since the PR was open in my tab in `files` section. @dharmit @bamachrn

**Pull request merged**